package.json

```diff
@@ -207,7 +207,8 @@
     "tslib": "^2.3.0",
     "viem": "^1.2.12",
     "wagmi": "^1.3.8",
-    "web3": "^1.8.0",
+    "web3": "^4.0.3",
+    "web3-utils-legacy": "npm:web3-utils@1.10.0",
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency